docker/set_up_ubuntu.sh

```diff
@@ -19,3 +19,20 @@ apt-get install -y gcc apt-transport-https build-essential python-dev \
         libsasl2-dev \
         libffi-dev \
         git
+
+# Stomp on the bash prompt with something more useful for development.
+cat > /etc/bash.bashrc <<EOF
+# Get the name for the current uid and drop error messages.
+MYUSER="\$(id -u -n 2> /dev/null)"
+
+# If id -u -n 2 didn't return a 0 exit code, then it's because there is
+# no name for that uid in the container. So set it to "you".
+if [[ "\$?" != "0" ]]; then
+    MYUSER="you"
```

Comment:
Technically we don't know if you're running with the uid/gid of a user on the host and we can't tell what the account name for that user is from inside the container. So I'm using "you" here, but maybe we should use a different indicator?

The other possible options are "root" and "app"--both of those are users that exist inside the container.

Reply:
"you" seems fine to me.